### DISCUSSION TITLE
how to return the id of the inserted record ?

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#12338] [Q&A] [answered]
I just created  a  document record in db, how could I get its id?

**Comment by ns8482e** [comment, marked as answer]
You can read property `int Id` on your document. If you dont have then create one

**Reply by scil** [reply]
Yes, it did work, even I not yet call  `await _session.FlushAsync()`

```
YesSql.ISession _sessin;

_session.Save(model);

model.Id

```
